manifests/init.pp

```diff
@@ -303,6 +317,13 @@
   Enum['none', 'plain', 'login', 'cram-md5'] $email_smtp_authentication = $::foreman::params::email_smtp_authentication,
   Optional[String] $email_smtp_user_name = $::foreman::params::email_smtp_user_name,
   Optional[String] $email_smtp_password = $::foreman::params::email_smtp_password,
+  Optional[String] $telemetry_prefix = $::foreman::params::telemetry_prefix,
+  Boolean $telemetry_prometheus_enabled = $::foreman::params::telemetry_prometheus_enabled,
+  Boolean $telemetry_statsd_enabled = $::foreman::params::telemetry_statsd_enabled,
+  Optional[String] $telemetry_statsd_host = $::foreman::params::telemetry_statsd_host
+  Optional[Enum['statsd', 'statsite', 'datadog']] $telemetry_statsd_protocol = $::foreman::params::telemetry_statsd_protocol
```

Comment:
I'd actually rather not add this, it can overwhelm users with options and we actually do not use any extensions, I think it might be better idea simply to remove this setting. Or at least write "keep statsd" in documentation comment.

Reply:
It's already hidden under advanced params. If we do not expose it and user changes protocol, on next upgrade, it would be overriden back. Not sure how you mean "keep statsd"